Packages/Tracking/Core/Runtime/Scripts/LeapImageRetriever.cs

```diff
@@ -602,16 +611,6 @@ private IEnumerator serviceCoroutine()
             controller.Disconnect += onDisconnect;
             controller.ImageReady += onImageReady;
             controller.DistortionChange += onDistortionChange;
-            _provider.OnDeviceChanged += OnDeviceChanged;
-        }
-
-        private void OnDeviceChanged(Device d)
```

Comment:
Can this be added back in? 
It handles the case when the specific serial number on the service provider gets changed at runtime. Without it, changing the serial number results in a frozen image.

Also the `unsubscribeFromService()` and `subscribeFromService()` was added in [this MR](https://gitlab.ultrahaptics.com/xr/integrations/unityplugin/-/merge_requests/267) to stop the image freezing when the service is restarted